src/tests/trips.test.js

```diff
@@ -18,9 +18,13 @@ describe('/trips/{ oneway | return }', () => {
     await db.room.destroy({ where: {}, force: true });
     await db.hotel.destroy({ where: {}, force: true });
     await db.user.destroy({ where: {}, force: true });
+    await db.request.destroy({ where: {}, force: true });
     await db.room.create(tripsData.rooms[0]);
     await db.room.create(tripsData.rooms[1]);
     await db.room.create(tripsData.rooms[2]);
+    await db.request.create(tripsData.requests[0]);
+
+    await db.trip.create(tripsData.trips[3]);
     await db.hotel.create(tripsData.hotels[0]);
```

Comment:
I think can use a script file called factories in `tests/script` that handles the database functionality on tests.

Reply:
let me try it out